addons/website_slides/tests/test_slide_slide.py

```diff
@@ -22,6 +22,43 @@ def test_slide_create_vote_constraint(self):
                 'vote': 2,
             })
 
+    def test_compute_category_completion_time(self):
+        """
+            Check that we properly calculate the completion time of a course without error, after deleting a slide.
+        """
+        self.category2 = self.env['slide.slide'].with_user(self.user_officer).create({
+            'name': 'Cooking Tips For Dieting',
+            'channel_id': self.channel.id,
+            'is_category': True,
+            'is_published': True,
+            'sequence': 5,
+        })
+        self.slide_4 = self.env['slide.slide'].with_user(self.user_officer).create({
+            'name': 'Vegan Diet',
+            'channel_id': self.channel.id,
+            'slide_category': 'document',
+            'is_published': True,
+            'completion_time': 5.0,
+            'sequence': 6,
+        })
+        self.slide_5 = self.env['slide.slide'].with_user(self.user_officer).create({
+            'name': 'Normal Diet',
+            'channel_id': self.channel.id,
+            'slide_category': 'document',
+            'is_published': True,
+            'completion_time': 1.5,
+            'sequence': 7,
+        })
+
+        before_unlink = self.category2.completion_time
+        self.assertEqual(before_unlink, 6.5)
```

Comment:
``self.slide_4.completion_time + self.slide_5.completion_time`` maybe so that it relies on test data ?